spring/spring4/spring4-embedded/src/test/java/org/infinispan/spring/support/embedded/InfinispanNamedEmbeddedCacheFactoryBeanTest.java

```diff
@@ -42,8 +42,6 @@
    public void startCacheManagers() {
       DEFAULT_CACHE_MANAGER = TestCacheManagerFactory.createCacheManager();
       DEFAULT_CACHE_MANAGER.start();
-      DEFAULT_CACHE_MANAGER.defineConfiguration("test.cache.Name", DEFAULT_CACHE_MANAGER.getDefaultCacheConfiguration());
-      DEFAULT_CACHE_MANAGER.defineConfiguration("test.bean.Name", DEFAULT_CACHE_MANAGER.getDefaultCacheConfiguration());
```

Comment:
These were already defined elsewhere in the test (causing an exception) and defining it as default does nothing anyways.